internal/provider/data_source_aci_relation_to_domain_test.go

```diff
@@ -23,9 +23,7 @@ func TestAccDataSourceFvRsDomAttWithFvAEPg(t *testing.T) {
 				Check: resource.ComposeAggregateTestCheckFunc(
 					resource.TestCheckResourceAttr("data.aci_relation_to_domain.test", "target_dn", "uni/vmmp-VMware/dom-domain_1"),
 					resource.TestCheckResourceAttr("data.aci_relation_to_domain.test", "annotation", "orchestrator:terraform"),
-					resource.TestCheckResourceAttr("data.aci_relation_to_domain.test", "binding_type", "none"),
```

Comment:
Why are we removing these?

Reply:
Introduced version_mismatch in default which takes care of this.